packages/opentelemetry-scope-async-hooks/src/AsyncHooksScopeManager.ts

```diff
@@ -58,7 +78,9 @@ export class AsyncHooksScopeManager implements ScopeManager {
     if (scope === undefined) {
       scope = this.active();
```

Comment:
What's the behaviour if this.active() is undefined ?

Reply:
Then the scope will be undefined and you'll get `undefined` when trying to get the current context